### DISCUSSION TITLE
kyuubi能否支持trino、hive的数据权限

### DISCUSSION THREAD

**Discussion by TracyMcgrady256** [apache/kyuubi#3889] [Q&A]
如题

**Comment by pan3793** [comment]
The Ranger has official Trino and Hive plugin, I don't think Kyuubi need to do that.

**Reply by TracyMcgrady256** [reply]
你好，能否说明下Trino、Hive如何用kyuubi+ranger控制权限吗？官网只有spark的权限控制部署文档

**Reply by pan3793** [reply]
The big data stack is composes of a bunch of the components, your question is tightly coupled w/ Hive/Trino and Ranger, but not Kyuubi, have you read those components documentation before finding solution in Kyuubi community?
